Assets/Scripts/ScriptableObjects/TestingControllers/PlayerCam.cs

```diff
@@ -0,0 +1,36 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerCam : MonoBehaviour
```

Comment:
Same as mouseLook, no real reason to make this from scratch unless you were doing something incredibly artisanal and still cinemachine handles a lot.

Reply:
Same as previous ^